docs/contributing/contribute.md

```diff
@@ -85,15 +85,15 @@ When you decide to start, this reading material is going to come handy:
 - [Mint your own DAI](https://medium.com/ethereum-grid/forking-ethereum-mainnet-mint-your-own-dai-d8b62a82b3f7)
 - [Dev Docs](https://docs.yearn.finance/developers/v2/getting-started)
 
-Once you have the strategy ready. This will guide you further down the road.
+Once you have the strategy ready, this will guide you further down the road:
```

Comment:
Once you have (a|your) strategy ready, ...

Reply:
```suggestion
Once you have your strategy ready, this will guide you further down the road:
```